applications/vessel-history/src/features/map/Map.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactElement, useState } from 'react'
+import { useSelector } from 'react-redux'
+import { InteractiveMap } from '@globalfishingwatch/react-map-gl'
+import { useLayerComposer } from '@globalfishingwatch/react-hooks'
+import {
+  getLayerComposerLayers,
+  // getMapboxPaintIcon,
+  // selectDirectionPointsLayers,
+  // selectLegendLabels,
+} from './map.selectors'
+import {
+  useGeneratorsConnect,
+  // useMapBounds,
+  // useMapClick,
+  // useMapMove,
+  // useViewport,
+} from './map.hooks'
+import { useMapboxRef, useMapboxRefCallback } from './map.context'
+import styles from './Map.module.css'
+
+const Map = (): ReactElement => {
+  const mapRef = useMapboxRef()
+
+  const onRefReady = useMapboxRefCallback()
+  // const { viewport, onViewportChange } = useViewport()
+
+  const generatorConfigs = useSelector(getLayerComposerLayers)
```

Comment:
We created a [dataview-client package](https://github.com/GlobalFishingWatch/frontend/blob/master/packages/dataviews-client/src/resolve-dataviews-generators.ts) to convert from dataviews to generators. Let's use it here to simplify the workflow